### DISCUSSION TITLE
Error: flag needs an argument: --config

### DISCUSSION THREAD

**Discussion by anup6191** [JanDeDobbeleer/oh-my-posh#5600] [Troubleshoot]
I had an issue when I ran this cmd "winget install JanDeDobbeleer.OhMyPosh -s winget" after restarting my PowerShell.

PowerShell 7.4.5
Error: flag needs an argument: --config
Usage:
  oh-my-posh prompt init [bash|zsh|fish|powershell|pwsh|cmd|nu|tcsh|elvish|xonsh] [flags]

Flags:
      --debug    enable/disable debug mode
  -h, --help     help for init
  -m, --manual   enable/disable manual mode
  -p, --print    print the init script
  -s, --strict   run in strict mode

Global Flags:
  -c, --config string   config file path

**Comment by lewis-yeung** [comment]
@anup6191 What's the initialization command in your PowerShell profile?
